utils/src/parallel_restore.h

```diff
@@ -68,6 +68,21 @@ struct scoutfs_parallel_restore_inode {
 	unsigned int target_len; /* not including null terminator */
 };
 
+struct scoutfs_parallel_restore_quota_rule {
+	u64 limit;
+	u8  prio;
+	u8  op;
+	u8  rule_flags;
+	struct quota_rule_name {
+		u64 val;
+		u8  source;
+		u8  flags;
+		u8  pad[7];
```

Comment:
This should be a different length padding, don't you agree? Moreover, `names` isn't aligned at all right now, so we'd want another pad in front to make that align. But I really wonder if you just can't use `struct scoutfs_ioctl_quota_rule` instead, which has all the correct padding.